lib/gooddata/models/blueprint/to_wire.rb

```diff
@@ -97,7 +97,10 @@ def self.dataset_to_wire(project, dataset)
             title: GoodData::Model.title(dataset),
             anchor: anchor_to_wire(project, dataset),
             attributes: attributes_to_wire(project, dataset),
-            facts: DatasetBlueprint.facts(dataset).map { |f| fact_to_wire(dataset, f) },
+            facts: DatasetBlueprint.facts(dataset).map do |f|
```

Comment:
I don't know why we should change it. I think we should keep the diff as small as we can, right?